### PR TITLE
Catch errors on failed DOI retrieval

### PR DESCRIPTION
**Pull request by JTeijema**
This PR will attempt to fix them. It involves JSON errors with the internet access functions. Ran across these issues using the script.

### CODE REVIEW

**Comment by JTeijema** [comment]
Schermafbeelding 2021-11-29 144934

For example

**Comment by JTeijema** [comment]
And, 
img_20211129_132345

Excuse the phone quality

**Comment by JTeijema** [comment]
I've also added a total arg to the tqdm, for extra styling points.

**Comment by valmelnikov** [comment]
I would suggest:
- outputting failing titles (at least for the sake of our understanding, why it is failed to be encoded)
- collecting count and outputting percentage of failing titles (to understand what is the success rate of retrieving DOI for non-failed titles)

File: scripts/crossref_doi_retrieval.ipynb [hunk 1]
```diff
@@ -73,22 +73,34 @@
     "    if not 1800 <= row.year <= 2021:\n",
     "        return None\n",
     "    year = str(int(row.year))\n",
-    "    url = 'https://api.crossref.org/works/?query.title=' + urllib.parse.quote(row.title) + \\\n",
+    "\n",
+    "    try: \n",
+    "        title = urllib.parse.quote(row.title)\n",
+    "    except:\n",
+    "        print(\"Title could not be encoded\")\n",
```
Comment (valmelnikov) [inline scripts/crossref_doi_retrieval.ipynb:80]: here it would be nice to increment the counter of encoding-failed titles

Comment (J535D165) [reply scripts/crossref_doi_retrieval.ipynb:80]: Fully agree

**Review by Rensvandeschoot — APPROVED** [review]

**Pull request merged**